core/sched.c

```diff
@@ -118,7 +118,14 @@ int __attribute__((used)) sched_run(void)
 #ifdef MODULE_SCHEDSTATISTICS
         schedstat *active_stat = &sched_pidlist[active_thread->pid];
         if (active_stat->laststart) {
-            active_stat->runtime_ticks += now - active_stat->laststart;
+            /* ensure now > laststart, to avoid flawed sum of runtime ticks
```

Comment:
mhm, how do you get 71 minutes? Doesn't that depend on the timer frequency - which is 1MHz for many boards but 250kHz for Ardunio Uno, for instance ...